src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
+    <PackageId>Microsoft.Azure.CognitiveServices.Language.SpellCheck</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
-    <VersionPrefix>1.2.0</VersionPrefix>
-    <AssemblyName>Microsoft.Azure.CognitiveServices.SpellCheck</AssemblyName>
+    <VersionPrefix>1.3.0-preview</VersionPrefix>
```

Comment:
Changing namespace name is a breaking change. You might want to rev the major version number (`2.0.0`)
@shahabhijeet FYI

Reply:
Done